project-generator/src/test/resources/io/syndesis/project/converter/test-pull-push-pom.xml

```diff
@@ -16,6 +16,7 @@
     <camel.version>@camel.version@</camel.version>
     <funktion.version>@funktion.version@</funktion.version>
     <atlasmap.version>@atlasmap.version@</atlasmap.version>
+    <syndesis-connectors.version>@syndesis-connectors.version@</syndesis-connectors.version>
```

Comment:
Well that test would work, but the parameter is used in the pom, so then that would be undefined. Maybe I'm missing something.